### DISCUSSION TITLE
Is there feature on livewire powergrid for fixed column at end or beginning but other content are scrollable.

### DISCUSSION THREAD

**Discussion by rohitshakyaa** [Power-Components/livewire-powergrid#1543] [General]
just like in tabulator frozen column feature
[Tabulator frozen col](https://tabulator.info/examples/6.2#frozen-col)

how can i implement such feature in livewire powergrid

**Comment by dansysanalyst** [comment]
Hi @rohitshakyaa,

Probably the best solution would be to create a [Custom Theme](https://livewire-powergrid.com/expanding-powergrid/custom-theme.html) to implement the required CSS classes.

One simpler approach, a first step that may be helpful, is applying Tailwind's [sticky](https://tailwindcss.com/docs/position#sticky-positioning-elements) class to your first column. 

```php
class MyTable extends PowerGridComponent
{
    public function columns(): array
    {
        return [

            Column::make('Barcode', 'barcode')
                ->bodyAttribute('sticky left-0  bg-white')
                 ->headerAttribute('sticky left-0 h-fit bg-gray-200'),

            Column::make('ID', 'id'),
            
            //....
        ];
    }
}

```



Resulting in:

![CleanShot 2024-05-12 at 12 53 25](https://github.com/Power-Components/livewire-powergrid/assets/79267265/6e3a81e4-320f-4144-8105-e5844afc36d7)
